doc/source/troubleshooting.rst

```diff
@@ -131,7 +131,7 @@ In no particular order, here's high-level advice that have proven to be useful i
 - The built-in Django development server provided by the default offline API client and ``ara-manage runserver`` is simple and convenient but it isn't meant to provide the best scalability and performance
 - The API server should be run as a service with a WSGI application server like gunicorn, uwsgi or mod_wsgi with apache2/httpd
 - There should be a frontend, reverse-proxy or load balancer such as apache, nginx, haproxy or traefik in front of the API server in order to handle TLS termination, caching and authentication
-- When :ref:`enabling authentication <api-security:Authentication and user management>`, consider using ``EXTERNAL_AUTH`` instead of the Django built-in user management to avoid a database lookup performance hit on every query
+- When :ref:`enabling authentication <api-security:Authentication and user management>`, consider using ``EXTERNAL_AUTH`` instead of the Django built-in user management to avoid the very significant performance hit of password hashing on every request
```

Comment:
Thanks for the clarification.

This isn't inaccurate but my understanding is that there is still a database lookup involved (it's where the password is stored). It's also OK to link to the issue for more context, thus I would suggest something like:

```suggestion
- When :ref:`enabling authentication <api-security:Authentication and user management>`, consider using ``EXTERNAL_AUTH`` instead of the Django built-in user management. This will avoid a significant performance hit incurred while hashing the password when authentication must be validated against the database backend. (See issue `#283 <https://github.com/ansible-community/ara/issues/283>`_)
```

Reply:
you're absolutely right, I didn't realize I completely dropped the hint about the database query, thank!